pkg/webhook/server.go

```diff
@@ -82,6 +86,74 @@ func (s *Server) setDefaults() {
 	}
 }
 
+// For registers defaulting and validation webhooks for the provided types that implement either
+// webhooktuil.validator or webhookutil.defaulter.
+func (s *Server) EnableDefaultsFor(scheme *runtime.Scheme, objects ...runtime.Object) error {
```

Comment:
we can make scheme injected from manager and get rid of this parameter ?

Reply:
Same comment as https://github.com/kubernetes-sigs/controller-runtime/pull/328#discussion_r259543868